datastore/google/cloud/datastore/transaction.py

```diff
@@ -186,7 +186,7 @@ def begin(self):
         try:
             self._id = self._client._connection.begin_transaction(
                 self.project)
-        except:
+        except:  # noqa: E722 do not use bare except, specify exception instead
```

Comment:
FWIW I assumed you'd summarize in your own words, but this fits on one line, so it's fine